### PR TITLE
Updated print spaces code quality changes for bitmessagecli.py

### PR DESCRIPTION
**Pull request by kdcis**
Removed spaces from print statements & % operator

### CODE REVIEW

File: src/bitmessagecli.py [hunk 1]
```diff
@@ -136,24 +136,24 @@ def apiInit(apiEnabled):
...
         uInput = userInput("Would you like to do this now, (Y)es or (N)o?").lower()
 
         if uInput == "y":  # User said yes, initalize the api by writing these values to the keys.dat file
-            print (' ')
+            print(' ')
 
             apiUsr = userInput("API Username")
             apiPwd = userInput("API Password")
             apiPort = userInput("API Port")
-            apiEnabled = userInput("API Enabled? (True) or (False)").lower()
```
Comment (PeterSurda) [inline src/bitmessagecli.py:148]: revert

File: src/bitmessagecli.py [hunk 2]
```diff
@@ -136,24 +136,24 @@ def apiInit(apiEnabled):
...
-            daemon = userInput("Daemon mode Enabled? (True) or (False)").lower()
+            apiEnabled = userInput("API Enabled?(True) or(False)").lower()
+            daemon = userInput("Daemon mode Enabled?(True) or(False)").lower()
 
             if (daemon != 'true' and daemon != 'false'):
-                print ('\n     Invalid Entry for Daemon.\n')
+                print('\n     Invalid Entry for Daemon.\n')
                 uInput = 1
```
Comment (PeterSurda) [inline src/bitmessagecli.py:153]: Maybe
```
uInput = "1"
```

Needs to be tested, but fixes code quality complaint.

File: src/bitmessagecli.py [hunk 3]
```diff
@@ -941,11 +942,11 @@ def inbox(unreadOnly=False):
                 messagesUnread += 1
 
         if messagesPrinted % 20 == 0 and messagesPrinted != 0:
-            userInput('(Press Enter to continue or type (Exit) to return to the main menu.)').lower()  # uInput =
```
Comment (PeterSurda) [inline src/bitmessagecli.py:944]: revert

File: src/bitmessagecli.py [hunk 4]
```diff
@@ -1236,9 +1239,9 @@ def buildKnownAddresses():
         addressBook = json.loads(response)
         for entry in addressBook['addresses']:
             if entry['address'] not in knownAddresses:
-                knownAddresses[entry['address']] = "%s (%s)" % (entry['label'].decode('base64'), entry['address'])
```
Comment (PeterSurda) [inline src/bitmessagecli.py:1239]: revert

File: src/bitmessagecli.py [hunk 5]
```diff
@@ -1605,33 +1608,33 @@ def UI(usrInput):
...
         msgNum = int(userInput("What is the number of the message you wish to open?"))
 
         if (uInput == 'i' or uInput == 'inbox'):
-            print ('\n     Loading...\n')
+            print('\n     Loading...\n')
             messageID = readMsg(msgNum)
 
             uInput = userInput("\nWould you like to keep this message unread, (Y)es or (N)o?").lower()
 
-            if not (uInput == 'y' or uInput == 'yes'):
```
Comment (PeterSurda) [inline src/bitmessagecli.py:1620]: revert

Comment (PeterSurda) [reply src/bitmessagecli.py:1620]: or even better refactor
```
if uInput not in ('y', 'yes'):
```

File: src/bitmessagecli.py [hunk 6]
```diff
@@ -1640,31 +1643,31 @@ def UI(usrInput):
...
             else:
-                print ('\n     Invalid entry\n')
+                print('\n     Invalid entry\n')
                 usrPrompt = 1
 
         elif (uInput == 'o' or uInput == 'outbox'):
             readSentMsg(msgNum)
 
             # Gives the user the option to delete the message
-            uInput = userInput("Would you like to (D)elete, or (Exit) this message?").lower()
```
Comment (PeterSurda) [inline src/bitmessagecli.py:1655]: revert

File: src/bitmessagecli.py [hunk 7]
```diff
@@ -1855,28 +1858,28 @@ def main():
...
-            print ('     Either Bitmessage is not running or your settings are incorrect.')
-            print ('     Use the command "apiTest" or "bmSettings" to resolve this issue.')
-            print ('     ****************************************************************\n')
+            print('\n     ****************************************************************')
+            print('        WARNING: You are not connected to the Bitmessage client.')
+            print('     Either Bitmessage is not running or your settings are incorrect.')
+            print('     Use the command "apiTest" or "bmSettings" to resolve this issue.')
+            print('     ****************************************************************\n')
 
-        print ('Type (H)elp for a list of commands.')  # Startup message)
```
Comment (PeterSurda) [inline src/bitmessagecli.py:1871]: revert second space, not first one

File: src/bitmessagecli.py [hunk 8]
```diff
@@ -554,25 +554,26 @@ def listAdd():
...
-    # print ('\nAddress Number,Label,Address,Stream,Enabled\n')
-    print ('\n     --------------------------------------------------------------------------')
-    print ('     | # |       Label       |               Address               |S#|Enabled|')
-    print ('     |---|-------------------|-------------------------------------|--|-------|')
+    # print('\nAddress Number,Label,Address,Stream,Enabled\n')
+    print('\n     --------------------------------------------------------------------------')
+    print('     | # |       Label       |               Address               |S#|Enabled|')
+    print('     |---|-------------------|-------------------------------------|--|-------|')
     for addNum in range(0, numAddresses):  # processes all of the addresses and lists them out
         label = (jsonAddresses['addresses'][addNum]['label']).encode(
```
Comment (PeterSurda) [inline src/bitmessagecli.py:566]: revert

**Review by PeterSurda — CHANGES_REQUESTED** [review]
Please change, most is ok.

File: src/bitmessagecli.py [hunk 9]
```diff
@@ -206,11 +206,11 @@ def apiData():
             BMConfigParser().get('bitmessagesettings', 'port')
         except:
```
Comment (PeterSurda) [inline src/bitmessagecli.py:207]: if you want, you can for example use
```except BaseException:```
or suppress warning

**Review by PeterSurda — CHANGES_REQUESTED** [review]

**Comment by g1itch** [comment]
Editing unmaintained scripts with no tests is a bad habit.

**Comment by kdcis** [comment]
> Editing unmaintained scripts with no tests is a bad habit.

Ya, @g1itch 
I'll pick further files which test cases already written.

**Comment by g1itch** [comment]
> > Editing unmaintained scripts with no tests is a bad habit.
> 
> Ya, @g1itch
> I'll pick further files which test cases already written.

Better to write new tests.

File: src/bitmessagecli.py (hunk 3 above)
Comment (kdcis) [reply src/bitmessagecli.py:944]: reverted.

File: src/bitmessagecli.py [hunk 10]
```diff
@@ -956,37 +957,39 @@ def outbox():
...
         # ]))
-        print (''.join([
+        print(''.join([
             '     Last Action Time:',
             datetime.datetime.fromtimestamp(
                 float(outboxMessages['sentMessages'][msgNum]['lastActionTime'])).strftime('%Y-%m-%d %H:%M:%S'),
         ]))
 
         if msgNum % 20 == 0 and msgNum != 0:
-            userInput('(Press Enter to continue or type (Exit) to return to the main menu.)').lower()  # uInput =
```
[the remark this answers is not in the document]
Comment (kdcis) [reply src/bitmessagecli.py:985]: reverted.

File: src/bitmessagecli.py (hunk 4 above)
Comment (kdcis) [reply src/bitmessagecli.py:1239]: reverted.

File: src/bitmessagecli.py (hunk 8 above)
Comment (kdcis) [reply src/bitmessagecli.py:566]: reverted.

File: src/bitmessagecli.py (hunk 9 above)
Comment (kdcis) [reply src/bitmessagecli.py:207]: Raised separate PR #1801 for it.

File: src/bitmessagecli.py [hunk 11]
```diff
@@ -1731,25 +1734,25 @@ def UI(usrInput):
                 if msgNum == 'a' or msgNum == 'all':
```
[the remark this answers is not in the document]
Comment (kdcis) [reply src/bitmessagecli.py:1733]: Will update it in separate PR.

File: src/bitmessagecli.py (hunk 2 above)
Comment (kdcis) [reply src/bitmessagecli.py]: Updated

File: src/bitmessagecli.py (hunk 1 above)
Comment (kdcis) [reply src/bitmessagecli.py]: Reverted

**Comment by PeterSurda** [comment]
Only spaces, revert all else.

**Comment by kdcis** [comment]
> Only spaces, revert all else.

Updated code to keep only spaces related changes & reverted others.

File: src/bitmessagecli.py [hunk 12]
```diff
@@ -149,11 +149,11 @@ def apiInit(apiEnabled):
             daemon = userInput("Daemon mode Enabled? (True) or (False)").lower()
 
             if (daemon != 'true' and daemon != 'false'):
-                print ('\n     Invalid Entry for Daemon.\n')
-                uInput = 1
```
Comment (PeterSurda) [inline src/bitmessagecli.py:153]: revert `uInput`

Comment (kdcis) [reply src/bitmessagecli.py:153]: reverted

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**